Source/Dafny/DafnyOptions.cs

```diff
@@ -979,146 +979,221 @@ public enum IncludesModes {
+
 /dprint:<file>
-    print Dafny program after parsing it
-    (use - as <file> to print to console)
+
+    Print the Dafny program after parsing it (use - as <file> to print
+    to the console)
+
 /rprint:<file>
-    print Dafny program after resolving it
-    (use - as <file> to print to console)
+
+    Print the Dafny program after resolving it (use - as <file> to print
+    to the console)
+
 /printMode:<Everything|DllEmbed|NoIncludes|NoGhost>
-    Everything is the default.
-    DllEmbed prints the source that will be included in a compiled dll.
-    NoIncludes disables printing of {{:verify false}} methods incorporated via the
-    include mechanism, as well as datatypes and fields included from other files.
-    NoGhost disables printing of functions, ghost methods, and proof statements in
-    implementation methods.  It also disables anything NoIncludes disables.
+
+    Everything is the default. DllEmbed prints the source that will be
+    included in a compiled dll. NoIncludes disables printing of
+    {{:verify false}} methods incorporated via the include mechanism, as
+    well as datatypes and fields included from other files. NoGhost
+    disables printing of functions, ghost methods, and proof statements
+    in implementation methods. It also disables anything NoIncludes
+    disables.
+
 /printIncludes:<None|Immediate|Transitive>
-    None is the default.
-    Immediate prints files included by files listed on the command line
-    Transitive recurses on the files printed by Immediate
-    Immediate and Transitive will exit after printing.
+
+    None is the default. Immediate prints files included by files listed
+    on the command line Transitive recurses on the files printed by
+    Immediate Immediate and Transitive will exit after printing.
+
 /view:<view1, view2>
-    print the filtered views of a module after it is resolved (/rprint).
-    If print before the module is resolved (/dprint), then everything in the module
-    is printed.
-    If no view is specified, then everything in the module is printed.
+
+    Print the filtered views of a module after it is resolved (/rprint).
+    If print before the module is resolved (/dprint), then everything in
+    the module is printed. If no view is specified, then everything in
+    the module is printed.
+
 /showSnippets:<n>
-    0 (default) - don't show source code snippets for Dafny messages
-    1 - show a source code snippet for each Dafny message
-/funcCallGraph Print out the function call graph.  Format is: func,mod=callee*
-/pmtrace      print pattern-match compiler debug info
-/titrace      print type-inference debug info
+
+    0 (default) - Don't show source code snippets for Dafny messages.
+    1 - Show a source code snippet for each Dafny message.
+
+/funcCallGraph
+
+    Print out the function call graph. Format is: func,mod=callee*
+
+/pmtrace
+
+    Print pattern-match compiler debug info.
+
+/titrace
+
+    Print type-inference debug info.
```

Comment:
Overall, this style will make the help output take much more space. Do you really want that?

Reply:
I would at least not have a blank line between the option and the start of text

Reply:
I find it more readable this way, but I think removing the blank line after the option name is reasonable.